doc/manuals/admin/management_api.md

```diff
@@ -28,3 +29,67 @@ curl --request PUT <host>:<port>/log/trace/t1-t2,t3-t4
 'PUT-requests' overwrite the previous log settings. So, in order to ADD
 a trace level, a GET /log/trace must be issued first and after that the
 complete trace string to be sent in the PUT request can be assembled.
+
+
+## Sempahores
+Another useful (especially if the broker stops responding correctly) REST API is
+the semaphore listing offered:
+
+
+```
+curl <host>:<port>/admin/sem
+```
+
+The response is a listing of information of all the broker's semaphores:
+
+```
+{
+    "alarmMgr": {
+        "status": "free"
+    },
+    "connectionContext": {
+        "status": "free"
+    },
+    "connectionSubContext": {
+        "status": "free"
+    },
+    "dbConnection": {
+        "status": "free"
+    },
+    "dbConnectionPool": {
+        "status": "free"
+    },
+    "logMsg": {
+        "status": "free"
+    },
+    "request": {
+        "status": "free"
+    },
+    "subCache": {
+        "status": "free"
+    },
+    "timeStat": {
+        "status": "free"
+    },
+    "transaction": {
+        "status": "free"
+    }
+}
+```
+
+Short explanation of the semaphores:
```

Comment:
Look to the render of this list ("View" buttom)... not very pretty, isn't it?

I'd suggest to put the semaphore name in bold type, then "," and next the description (lowercase after ",").